modules/grafana/src/main/java/org/testcontainers/grafana/LgtmStackContainer.java

```diff
@@ -0,0 +1,56 @@
+package org.testcontainers.grafana;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.DockerImageName;
+
+/**
+ * Testcontainers implementation for Grafana Otel LGTM.
+ * <p>
+ * Supported image: {@code grafana/otel-lgtm}
+ * <p>
+ * Exposed ports:
+ * <ul>
+ *     <li>Grafana: 3000</li>
+ *     <li>Otel Http: 4317</li>
+ *     <li>Otel Grpc: 4318</li>
+ *     <li>Prometheus: 9090</li>
+ * </ul>
+ */
+public class LgtmStackContainer extends GenericContainer<LgtmStackContainer> {
```

Comment:
Let's log a _clickable_ HTTP url to the Grafana Web UI on startup (a bit similar to how we do it for VNC container).